sign/rpmgensig.cc

```diff
@@ -766,6 +772,13 @@ static int rpmSign(const char *rpm, int deleting, int flags)
 		/* Identical signature is not an error */
 		res = 0;
 	    }
+	    if (res == 2) {
+		rpmlog(RPMLOG_ERR,
+		   _("%s already contains legacy signature\n"),
```

Comment:
There's a missing "a" from legacy. Also I think it wouldn't hurt to clarify the legacy part: 

"already contains a legacy v3/v4 signature"

Reply:
I've left it out intentionally because the above warning "already contains identical signature, skipping". As in, program output, especially log messages, often leave out articles to be more concise, BUT - that's too much talk, I'll just add it :laughing:

Reply:
Oh. Maybe fix the other message while at it? :laughing:

Reply:
Indeed :smile: